packaging/ansible-runner-service-project/project/roles/gluster-replace-peers/tasks/peers.yml

```diff
@@ -2,49 +2,48 @@
 - name: Validating the state of the old host
   block:
     - name: Fetch the UUID of the old node
-      shell: >
+      ansible.builtin.shell: >
         gluster peer status |
         grep -A1 {{ gluster_old_node | mandatory }} |
         awk -F: '/uid/ { print $2}'
       register: uuid
-      tags:
-        - skip_ansible_lint # E301
+      changed_when: uuid.rc != 0
 
     - name: Get the state from peer file
-      shell: grep state "/var/lib/glusterd/peers/{{ uuid.stdout | trim }}"
+      ansible.builtin.command: grep state "/var/lib/glusterd/peers/{{ uuid.stdout | trim }}"
```

Comment:
Probably not a problem here but I think it's safer to use `argv` argument of `command` in situations with generated command line arguments.